### DISCUSSION TITLE
Latest macOS version of DXVK no longer works on Catalina

### DISCUSSION THREAD

**Discussion by Gravarty** [The-Wineskin-Project/WineskinServer#270] [General]
Trying to run DirectX 11 apps using DXVK on macOS Catalina the process crashes with this error:

0128:fixme:seh:virtual_unwind calling personality routine in system library not supported yet
wine: Unhandled page fault on read access to 0000000000000018 at address 00007FFF6AAC281D

This happened to all games i tried to run. Big Sur might be the minimum required version for DXVK now.

**Comment by Gcenx** [comment]
<img width="1268" alt="Screen Shot 2022-06-20 at 11 35 47 PM" src="https://user-images.githubusercontent.com/38226388/174805649-f2f42b3b-d48f-4cb4-9321-199f28e1e253.png">

As can be seen in this screenshot I've managed to launch AOE2 DE on macOS Mojave that has an Intel GPU.

**Reply by Gravarty** [reply]
Huh? Weird, i tried GTA V, Quake Remaster, Mafia Definitive Edition & Mafia II Definitive Edition and all of them failed with the same error message. Running the same Wineskin app on Monterey and Ventura works without issues.

**Reply by Gcenx** [reply]
To tested I’d created a new wrapper using the latest winecx engine and installed steam and the latest DXVK-macOS verb via winetricks.

I know that system won’t be able to launch GTAV but Quake Remaster should be possible so I’ll test that also once home from work.

However the single line message you’ve provided isn’t DXVK complaining but wine.

**Reply by Gravarty** [reply]
Oh okay, yeah, it's just weird that these messages only appeared on DX11/DXVK games. DX9/OpenGL games ran fine without these messages in the log.
